src/store/modules/accounts/ledger.js

```diff
@@ -70,24 +70,23 @@ export default {
     async signTransaction({ rootGetters: { account, 'sdkPlugin/sdk': sdk }, dispatch }, { txBase64 }) {
       await dispatch('ensureCurrentAccountAvailable');
 
-      const txObject = TxBuilder.unpackTx(txBase64).tx;
-      const binaryTx = TxBuilder.buildTx(
+      const txObject = unpackTx(txBase64).tx;
```

Comment:
```suggestion
      const txObject = unpackTx(txBase64);
```

Reply:
Resolving as this will be part of a separate PR